app/services/find_out_of_sync_deploys_service.rb

```diff
@@ -0,0 +1,35 @@
+class FindOutOfSyncDeploysService
+  class << self
+    delegate :call, to: :new
+  end
+
+  def call
+    return {} if out_of_sync_apps.empty?
+
+    out_of_sync_apps_info.group_by { |app| app[:dependencies_team] }
```

Comment:
I was undecided on this one so happy to go with `team`. `dependencies_team` [falls back to `team`](https://github.com/alphagov/govuk-developer-docs/blob/main/app/repo.rb#L168) if it's not set so my initial thinking was that it would be a bit more configurable should the team choose to have the notifications sent to some other channel.